toolkit/tools/srpmpacker/srpmpacker.go

```diff
@@ -758,10 +765,18 @@ func packSingleSPEC(specFile, srpmFile, signaturesFile, buildDir, outDir, distTa
 		return
 	}
 
-	// Hydrate all sources. Download any missing ones not in `sourceDir`
-	err = hydrateFiles(fileTypeSource, specFile, workingDir, srcConfig, currentSignatures, defines)
-	if err != nil {
-		return
+	{
+		// Limit the number of concurrent network operations by pushing a struct{} into the channel. This will block until
+		// another operation completes and removes the struct{} from the channel.
+		netOpsSemaphor <- struct{}{}
+		// Hydrate all sources. Download any missing ones not in `sourceDir`
+		err = hydrateFiles(fileTypeSource, specFile, workingDir, srcConfig, currentSignatures, defines)
+		if err != nil {
+			<-netOpsSemaphor
+			return
+		}
+		// Clear the channel to allow another operation to start
+		<-netOpsSemaphor
```

Comment:
Why the separate scope with `{}`?

Reply:
Left over from previous experiment with scoped variables, cleaned up.